WordPress/Classes/Models/AbstractPost+Searchable.swift

```diff
@@ -76,9 +76,9 @@ fileprivate extension AbstractPost {
             title = postTitle
         }
 
-        guard status != .publish, let statusTitle = statusTitle else {
+        guard status != .publish, let status else {
             return title
         }
-        return "[\(statusTitle)] \(title)"
```

Comment:
I haven't tested this part, but it's just moving code around, so it seems safe.

Reply:
I checked it and it's used for indexing spotlight search

![Simulator Screenshot - iPhone 15 - 2024-06-10 at 10 53 44](https://github.com/wordpress-mobile/WordPress-iOS/assets/4062343/2602081d-bb3e-4923-aefe-b62a67f01d20)